### PR TITLE
Document how to clear the built CSS/JS assets

### PR DESCRIPTION
**Pull request by BenSturmfels**

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -80,6 +80,9 @@ reloading, run `bin/webpack-dev-server` in a second console. Now after making a
 change to the JavaScript code you should see Webpack immediately rebuild the
 assets and perform a full page reload.
 
+When in doubt, you can manually clear the generated CSS and JavaScript files,
+run `rm -rf public/assets public/packs`.
```
Comment (wwahammy) [inline docs/getting_started.md:84]: There's actually a [built in command ](https://guides.rubyonrails.org/v6.0/command_line.html#rails-assets) for that: `bin/rails assets:clobber`. That's the appropriate way to handle that.

**Review by wwahammy — CHANGES_REQUESTED** [review]
Thanks @BenSturmfels. There's actually a built-in command for this as described.

File: docs/getting_started.md (hunk 1 above)
Comment (BenSturmfels) [reply docs/getting_started.md]: Thanks Eric, I've made that change.

**Review by wwahammy — APPROVED** [review]
Thanks @BenSturmfels!

**Pull request merged**